resourcet/Control/Monad/Trans/Resource.hs

```diff
@@ -268,7 +269,26 @@ resourceForkIO (ResourceT f) = ResourceT $ \r -> L.mask $ \restore ->
             (stateCleanup ReleaseException r)
             (restore $ f r))
 
-
+-- | This is identical to @resourceForkIO@, except that it allows the user
+-- to specify a function other than @forkIO@ (e.g., @async@) to create
+-- the thread.
+--
+-- Since 1.1.9
```

Comment:
I realize the other code doesn't do this, but can you `s/Since/@since`? Also, can you update the .cabal file?